### PR TITLE
Added Slowpoke

### PR DESCRIPTION
**Pull request by KarasIlia**

### CODE REVIEW

**Comment by swagat5147** [comment]
last cell is for evolution form, update it

File: index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h1>Pokemon Chart</h1>
 		<td><img src="https://vignette.wikia.nocookie.net/pokemon/images/7/70/079Slowpoke.png/revision/latest/scale-to-width-down/200?cb=20140328203941"></td>
 		<td>Slowpoke</td>
 		<td>Water</td>
-		<td><a href="https://pokemon.fandom.com/wiki/Slowpoke">Slowpoke</a></td>
+		<td><a href="https://pokemon.fandom.com/wiki/Slowbro">Slowpoke</a></td>
```
Comment (swagat5147) [inline index.html]: change the text too

Comment (KarasIlia) [reply index.html]: My bad

**Comment by swagat5147** [comment]
cool

**Pull request merged**